src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/ApplicationInformationController.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.controller
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import org.slf4j.LoggerFactory
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.ModelAttribute
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.bind.annotation.SessionAttributes
+import uk.gov.justice.digital.hmpps.pecs.jpc.domain.price.Supplier
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.AnnualPriceAdjustmentsService
+
+@RestController
+@RequestMapping(name = "Supplier information", path = ["/app"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@SessionAttributes(SUPPLIER_ATTRIBUTE)
+class ApplicationInformationController(val annualPriceAdjustmentsService: AnnualPriceAdjustmentsService) {
```

Comment:
The controller is fairly simplistic in terms of the information it provides.  This is the only requirement at the moment.  It can always be changed subject to future requirements coming in.